controllers/user.controller.js

```diff
@@ -1,12 +1,16 @@
-const Users = require('../models/user.model');
+// const Users = require('../models/user.model');
```

Comment:
Not sure why you just commented the address to the file. Since the file will not be used, would it not be a waste to have it present in the repository.

Reply:
That was a over sight, I had to check if your code was working first, before i deleted mine.

Reply:
That was a over sight, I had to check if your code was working first, before I deleted mine.